src/main/java/com/marklogic/mgmt/security/AmpManager.java

```diff
@@ -79,6 +79,33 @@ protected String getIdFieldName() {
         return params.toArray(new String[] {});
     }
 
+    /**
+     * In ML 8.0-5.1 deleting an amp fails if the database is specified on the
```

Comment:
I have an open ticket for this. The bug I ran into is that the Management API doesn't recognize a database for an amp. This is an issue on one of my projects, where we can never delete an amp. Does this fix that problem?